woocommerce-gutenberg-products-block.php

```diff
@@ -342,19 +342,24 @@ function wgpb_print_script_settings() {
 	$code = get_woocommerce_currency();
 
 	// Settings and variables can be passed here for access in the app.
+	// Will need `wcAdminAssetUrl` if the ImageAsset component is used.
+	// Will need `dataEndpoints.countries` if Search component is used with 'country' type.
+	// Deliberately excluding: `embedBreadcrumbs`, `trackingEnabled`.
 	$settings = array(
-		'adminUrl'   => admin_url(),
-		'wcAssetUrl' => plugins_url( 'assets/', WC_PLUGIN_FILE ),
-		'siteLocale' => esc_attr( get_bloginfo( 'language' ) ),
-		'currency'   => array(
+		'adminUrl'      => admin_url(),
+		'wcAssetUrl'    => plugins_url( 'assets/', WC_PLUGIN_FILE ),
+		'siteLocale'    => esc_attr( get_bloginfo( 'language' ) ),
+		'currency'      => array(
 			'code'      => $code,
 			'precision' => wc_get_price_decimals(),
 			'symbol'    => get_woocommerce_currency_symbol( $code ),
+			'position'  => get_option( 'woocommerce_currency_pos' ),
 		),
-		'date'       => array(
-			'dow' => get_option( 'start_of_week', 0 ),
-		),
-		'l10n'       => array(
+		'orderStatuses' => format_order_statuses( wc_get_order_statuses() ),
```

Comment:
Looks like this is a wc-admin function? It should probably have a `wc_*` prefix to avoid conflicts... and you'll need to check if it exists before using it here:

> Fatal error: Uncaught Error: Call to undefined function format_order_statuses() in /woocommerce-gutenberg-products-block.php:358

I think you can just set this to an empty array & add a note that if we use the `OrderStatus` component we'll need to give it real data. I don't see the need for that component in the blocks, though.

Reply:
Removed the key in 892a3aa. (It's only used for the `OrderStatus` component)